src/plang/TokenStream.cpp

```diff
@@ -6,26 +6,17 @@ using namespace plang;
 
 static const char* kOpTokenStrings[] =
 {
-//	arithmetic
+	//	arithmetic
 	"=", "+", "-", "/", "*", 
-//	control
+	//	control
 	":", "(", ")", "{", "}",
 };
 
-////////////////////////////////////////////////////////////////////////////////
 static bool isWhiteSpace(char _c)
 {
-	//static const char* kWhitespace = " \t\n";
-	//for (const char* it = kWhitespace; *it != 0; ++it) {
-	//	if (_c == *it) {
-	//		return true;
-	//	}
-	//}
-	//return false;
-	return _c <= ' '; // optim, just assume all ASCII control chars are whitespace
```

Comment:
Maybe keep this one, since the function is doing something a little bit weird I think it's nice to have a rationale for the weirdness in a comment.